crt_portal/cts_forms/templates/forms/widgets/fuzzy.html

```diff
@@ -1,7 +1,21 @@
 {% spaceless %}
   {% for widget in widget.subwidgets %}
+    {% if widget.name == 'location_name_1' %}
+      <p class="text-bold border-top margin-top-05 padding-top-2">
+        Refine search:
+      </p>
+    {% endif %}
     <div>
-      <label id="{{widget.name}}_label" for="id_{{widget.name}}">{{widget.attrs.label}}</label>
+      {% if widget.attrs.tooltip != null %}
```

Comment:
nit: consider switching these around so that the "if null" edge case is visible right next to its conditional.

It'll help avoid issues like this:

<img width="675" alt="image" src="https://github.com/usdoj-crt/crt-portal/assets/15126660/49a2b385-d708-452e-9eae-9ffb034678f7">